test/adapter/json/nested_relationships_test.rb

```diff
@@ -0,0 +1,203 @@
+require 'test_helper'
+
+NestedPostSerializer = Class.new(ActiveModel::Serializer) do
+  attributes :id, :title
+
+  has_many :comments, include: :author
+end
```

Comment:
I know this is the pattern in the poros.. but can we just do regular Ruby class declarations here, unless there's a good reason and we can comment that?